### PR TITLE
Make guild npcs interactable and add Guild_Open handler

### PR DESCRIPTION
**Pull request by sorokya**
Also adds some resource IDs for guild icons and text

### CODE REVIEW

File: EOLib/Domain/Interact/Guild/GuildActions.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace EOLib.Domain.Interact.Guild
+{
+    [AutoMappedType]
+    public class GuildActions : IGuildActions
+    {
+        private readonly IGuildSessionProvider _GuildSessionProvider;
+        private readonly IPacketSendService _packetSendService;
+
+        public GuildActions(IGuildSessionProvider GuildSessionProvider,
```
Comment (ethanmoffat) [inline EOLib/Domain/Interact/Guild/GuildActions.cs:13]: nit: lowercase the identifier name

File: EndlessClient/Dialogs/ActiveDialogRepository.cs [hunk 2]
```diff
@@ -144,6 +146,8 @@ public class ActiveDialogRepository : IActiveDialogRepository, IActiveDialogProv
 
         public Option<BarberDialog> BarberDialog { get; set; }
         
+        public Option<GuildDialog> GuildDialog { get; set; }
+
         public Option<ScrollingListDialog> HelpDialog { get; set;  }
```
Comment (ethanmoffat) [inline EndlessClient/Dialogs/ActiveDialogRepository.cs:151]: You need to also add the `.Map` to the `ActiveDialogs` collection and the `Dispose` resetting it below

File: EOLib/Domain/Interact/MapNPCActions.cs [hunk 3]
```diff
@@ -77,6 +77,13 @@ public void RequestBarber(NPC.NPC npc)
         }
     }
 
+        public void RequestGuild(NPC.NPC npc)
+        {
+            var packet = new GuildOpenClientPacket { NpcIndex = npc.Index };
+            _packetSendService.SendPacket(packet);
+        }
+    }
```
Comment (ethanmoffat) [inline EOLib/Domain/Interact/MapNPCActions.cs:85]: F in chat

**Review by ethanmoffat — CHANGES_REQUESTED** [review]

File: EOLib/Domain/Interact/MapNPCActions.cs (hunk 3 above)
Comment (sorokya) [reply EOLib/Domain/Interact/MapNPCActions.cs:85]: ?

Comment (ethanmoffat) [reply EOLib/Domain/Interact/MapNPCActions.cs:85]: Compiler error presumably due to merge

**Review by ethanmoffat — APPROVED** [review]

**Pull request merged**